src/sexp_internal.rs

```diff
@@ -13,10 +13,7 @@ pub fn display_cil(expr: &sexp::Sexp) -> String {
         Sexp::List(l) => {
             format!(
                 "({})",
-                l.iter()
-                    .map(display_cil)
-                    .collect::<Vec<String>>()
-                    .join(" ")
+                l.iter().map(display_cil).collect::<Vec<String>>().join(" ")
```

Comment:
Could you please squash this into the previous commit that changes the map call?